templates/test/cluster-template-prow-ha.yaml

```diff
@@ -170,52 +168,52 @@ spec:
           runtime-request-timeout: 5m
         name: '{{ ds.meta_data.name }}'
     postKubeadmCommands:
-    - mkdir -p /home/metal3/.kube
-    - cp /etc/kubernetes/admin.conf /home/metal3/.kube/config
+    - mkdir -p /home/${IMAGE_USERNAME}/.kube
+    - cp /etc/kubernetes/admin.conf /home/${IMAGE_USERNAME}/.kube/config
     - systemctl enable --now keepalived
-    - chown metal3:metal3 /home/metal3/.kube/config
+    - chown ${IMAGE_USERNAME}:${IMAGE_USERNAME} /home/${IMAGE_USERNAME}/.kube/config
     preKubeadmCommands:
     - netplan apply
     - systemctl enable --now crio kubelet
-    - if (curl -sk --max-time 10 https://192.168.111.249:6443/healthz); then echo "keepalived already running";else systemctl start keepalived; fi
+    - if (curl -sk --max-time 10 https://${API_ENDPOINT_HOST}:${API_ENDPOINT_PORT}/healthz); then echo "keepalived already running";else systemctl start keepalived; fi
```

Comment:
I think this should be `CLUSTER_APIENDPOINT_HOST` and `CLUSTER_APIENDPOINT_PORT` to keep the same variable names as are used in the metal3-dev-env.

Reply:
Good catch. I changed it elsewhere in the file but not here